cockpit/src/subscriptions-client.js

```diff
@@ -449,7 +449,7 @@ client.unregisterSystem = () => {
 
 function statusUpdateFailed(reason) {
     console.warn("Subscription status update failed:", reason);
-    client.subscriptionStatus.status = (reason && reason.problem) || "not-found";
+    client.subscriptionStatus.status = (reason && reason.problem) || _("Not-found");
```

Comment:
I am not sure why this is hypenated. Is there a requirement for a single word? If not, it is not gramatically correct to have it here.